source/getting_started/aiot_intelligent_edge/coco_for_applications/analytics_client_applications/browser_js_analytics_client_applications/pre_install_checklist_browser_js_analytics_client_apps.rst

```diff
@@ -3,9 +3,9 @@
 Pre-Install Checklist
 =====================
 
-.. figure:: /_static/coming-soon.png
-   :align: center
-
-   **Coming Soon**
-
 .. sectionauthor:: Deekshith
+
+1. Git
+2. node (**Recommended to use version 16.0.0**)
+3. npm
+4. angular cli (**npm install -g @angular/cli**)
```

Comment:
I suggest providing the command to run to install each of the checklist items

Reply:
handled